taipy/gui/hook.py

```diff
@@ -10,6 +10,7 @@ class Hook:
 
 
 class Hooks(object, metaclass=_Singleton):
+    """NOT DOCUMENTED"""
```

Comment:
I think @dinhlongviolin1 plans to change this to `class _Hooks()`

Reply:
Indeed.